tsconfig.json

```
@@ -7,6 +7,7 @@
       "dom.iterable",
       "esnext"
     ],
+    "incremental": true,
```

Biome (2.4.16)

[error] The key incremental was already declared.

(lint/suspicious/noDuplicateObjectKeys)